index.js

```diff
@@ -118,7 +118,7 @@ const stop = Symbol('stop');
 
 class HTTPError extends Error {
 	constructor(response) {
-		super(response.statusText);
+		super(response.statusText || response.status || 'Unknown response error');
```

Comment:
`.status` is a number, right? If so, it should be coerced to a string.

Reply:
The Error constructor handles coercion. But maybe TypeScript doesn't account for that?

Reply:
So why didn’t it handle coercion from undefined?

Reply:
Good question. I'm not entirely sure. Normally, `Error` coerces `undefined` to an empty string.

Maybe New Relic overwrites the global `Error` constructor to make it more strict? Hopefully, it would still accept numbers, though.

Reply:
> `.status` is a number, right? If so, it should be coerced to a string.

I can add string coercion. 

> So why didn’t it handle coercion from undefined?

Throwing `new Error()` with an argument would also result in an undefined message so it's not an invalid call.

> Maybe New Relic overwrites the global `Error` constructor to make it more strict? 

I think it's just a New Relic bug. It should be handling errors with undefined message properties.